sources/packages/web/src/views/institution/student/InstitutionStudentApplicationView.vue

```diff
@@ -0,0 +1,78 @@
+<template>
```

Comment:
just wondering, do we need to make this page a component? dev thoughts?

Reply:
I see the point @ann-aot but I am not sure since the main part of this page (IMO) is the `StudentApplication` that is already a component.

Reply:
ya, I was also having a double thought